controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -173,8 +173,7 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	// Check namespace is not exist, then create
 	namespace := instance.Spec.ApplicationsNamespace
-	err := r.createOdhNamespace(ctx, instance, namespace, platform)
-	if err != nil {
+	if err := r.createAppNamespace(ctx, instance, namespace, platform); err != nil {
```

Comment:
we probably need to perform some validation here:

1. there should be only one namespace with label `opendatahub.io/application-namespace`
2. if an namespace with label `opendatahub.io/application-namespace` exists, then its name must mach the one defined in the `instance.Spec.ApplicationsNamespace`

And report the error as part of the DSCI status.

Reply:
yep, i added the valiation within the createAppNamespace()

but i think "there should be only one namespace with label opendatahub.io/application-namespace" might not be needed.
say, user has 2 namespaces (e.g A and B) both with this label.  In DSCI only A is set for the applicationnamespace, that should be a valid use case, right? surely it is a waste for operator to have "B" but do no harm.

Reply:
It depends, i.e. if the user start adding things to that namespace, it may harm the operator and may introduce side effects (cache may grow, useless event are being received by the operator)

It also seems misleading to allow such case to be honest and may be confusing. 

Since it is an unsupported configuration, I don't see why we would need to support it

Reply:
ok we can limit that to only one namespace for now